tests/unit/disco/sagas/testDiscovery.js

```diff
@@ -0,0 +1,104 @@
+import SagaTester from 'redux-saga-tester';
+
+import { loadEntities } from 'core/actions';
+import { ErrorHandler } from 'core/errorHandler';
+import addonsReducer from 'core/reducers/addons';
+import apiReducer from 'core/reducers/api';
+import * as api from 'disco/api';
+import { getDiscoResults, loadDiscoResults } from 'disco/actions';
+import discoResultsReducer from 'disco/reducers/discoResults';
+import discoverySaga from 'disco/sagas/discovery';
+import { dispatchSignInActions } from 'tests/unit/amo/helpers';
+import {
+  createFetchDiscoveryResult, fakeDiscoAddon,
+} from 'tests/unit/disco/helpers';
+
+
+describe('disco/sagas/discovery', () => {
```

Comment:
oh man! I just realized that since Jest runs in Node (unlike our old karma suite) we can totally use `__filename`. Check it:

```diff
diff --git a/tests/unit/disco/sagas/testDiscovery.js b/tests/unit/disco/sagas/testDiscovery.js
index 3b98cf7a..9ee16c8b 100644
--- a/tests/unit/disco/sagas/testDiscovery.js
+++ b/tests/unit/disco/sagas/testDiscovery.js
@@ -14,7 +14,7 @@ import {
 } from 'tests/unit/disco/helpers';


-describe('disco/sagas/discovery', () => {
+describe(__filename, () => {
   describe('fetchDiscoveryAddons', () => {
     let apiState;
     let errorHandler;
```

<img width="666" alt="screenshot 2017-08-11 10 23 57" src="https://user-images.githubusercontent.com/55398/29219966-3bd7338c-7e7f-11e7-9afd-5d66b4b93fe9.png">